.github/workflows/ci.yml

```diff
@@ -141,7 +141,7 @@ jobs:
             ls ../PyInstaller/bootloader/Windows-64bit-arm
 
       - name: Update pip
-        run: python -m pip install --upgrade pip setuptools build
+        run: python -m pip install --upgrade pip setuptools hatchling
```

Comment:
Strictly speaking, setuptools is no longer needed for setup but I left it here in case the Python environments come with an older version of setuptools which would not be upgraded without an explicit `pip install --upgrade pip setuptools` somewhere which could delay discovering a new setuptools regression. I'm thinking perhaps it should instead be tracked `tests/requirements-libraries.txt`?

Reply:
Yes, it makes sense to move `setuptools` into `tests/requirements-libraries.txt`. This will also make it easier to identify issues that are linked to a new release of `setuptools`.